lua/autorun/wire_load.lua

```diff
@@ -63,21 +63,21 @@ if SERVER then
 	AddCSLuaFile("wire/client/text_editor/modes/zcpu.lua")
 
 	-- HL-ZASM
-	AddCSLuaFile("wire/client/hlzasm/hc_compiler.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_opcodes.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_expression.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_preprocess.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_syntax.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_codetree.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_optimize.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_output.lua")
-	AddCSLuaFile("wire/client/hlzasm/hc_tokenizer.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_compiler.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_opcodes.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_expression.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_preprocess.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_syntax.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_codetree.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_optimize.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_output.lua")
+	AddCSLuaFile("wire/zyelios/hlzasm/hc_tokenizer.lua")
```

Comment:
That being said, you did implement exactly what I suggested in the previous pull request - sorry for being so inconsistent :stuck_out_tongue_winking_eye: